recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ source:
     - patches/0001-Merge-pull-request-183-from-boostorg-ppc64le.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
```suggestion
  number: 0
```

Reply:
Fine with me, I was just blindly following the checklist.